### ISSUE TITLE
11/22 Meeting Notes

### ISSUE THREAD

**Issue by panalluri** [ocean-transport/air-sea-ML#12]
- CNN addition problem 
    - Increase amount of training data by a lot
    - Move all the constant functions in CNN training into a .py file (or more than one eventually -- e.g. training dataset creation, model architecture setup, model evaluation, etc)
- CNN complex arithmetic problem
    - same as above
- Super-resolution problem
    - Super-resolve only AeroBulk input values, not the ones I got off my function analysis 
    - Compare HR and LR images for each input
    - Use a much larger set of training data
    - Very different magnitudes for each input problem --> try normalizing beforehand
       - divide everything by order of magnitude (e.g. mean, std) so values end up around 1 
   - Plot loss curves for training and testing data together
- AeroBulk ANN replacement 
    - Increase training data size

- General NN trials
    - How do lr and normalization relate?
        - Multiply targets by N --> see if training losses differ for different Ns
        - Read online about this
    - For CNNs, increase batch size (eventually)

- Keeping track of different NN training experiments
    - Weights and biases (package and website)

- Set up Zotero

- Update diagnostics proposal

**Comment by panalluri** [comment]
Notes:
- Zotero will. be a future endeavor, when I am less busy with school
- Weights and biases will be something I look more into after finals chill
    - Learning curve to be tackled (e.g. version control for AI, etc)
- Did not trial the lr and normalization relation, but did look it up
